src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -561,6 +562,7 @@ public virtual PSCredential Credential
         /// to override the policy setting
         /// </remarks>
         [Parameter(ParameterSetName = PSRemotingBaseCmdlet.ComputerNameParameterSet)]
+        [Parameter(ParameterSetName = PSRemotingBaseCmdlet.SSHHostParameterSet)]
```

Comment:
I am a little worried about sharing parameters between WinRM and SSH parameter sets.  My worry is that PowerShell parameter resolution might resolve to the wrong parameter set for positional parameters (e.g., New-PSSession localhost -Port 8080).  Please verify that parameter set resolution does not change (i.e. causes a breaking change to existing behavior).